### PR TITLE
Chuli/fix resize group reflow

### PR DESCRIPTION
**Pull request by rachelcl9**
Temp PR to merge with @micahgodbolt 's command bar refactor work.
=============================
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

(give an overview)

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/CommandBar/CommandBarPage.tsx [hunk 1]
```diff
@@ -44,45 +44,10 @@ export class CommandBarPage extends React.Component<IComponentDemoPageProps, {}>
...
-        dos={
-          <div>
-            <ul>
-              <li>Sort commands in order of importance from left to right or right to left depending on the culture.</li>
-              <li>Organize commands into logical groupings.</li>
-              <li>Display no more than 5-7 commands.</li>
-              <li>Use overflow to house less frequently-used commands.</li>
-              <li>In small breakpoints, only have the most recognizable commands render as icon only.</li>
-            </ul>
+          <div>1
```
Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/CommandBar/CommandBarPage.tsx]: typo?

**Comment by micahgodbolt** [comment]
These changes are primarily for the resize group and overflow set. Can you create a PR to master with all the changes except those affecting commandBar files. Also be aware of #2112 which will be making some resizeGroup changes as well. Want to make sure the 2 PRs don't conflict.

File: packages/office-ui-fabric-react/src/components/CommandBar/CommandBarPage.tsx (hunk 1 above)
Comment (rachelcl9) [inline packages/office-ui-fabric-react/src/components/CommandBar/CommandBarPage.tsx]: I'll uncomment these before push to master. I was trying to test only one of the example to see if the synchronous rendering happens.

**Comment by micahgodbolt** [comment]
Closing this up for now as changes to resize group make this out of date.

**Pull request closed without merging**